README.md

```diff
@@ -2,61 +2,30 @@
 
```

Comment:
- [ ] Pull Request should have a descriptive title telling you what changes are inside, kindly implement this so the PR looks more professional